Dockerfile

```diff
@@ -1,53 +1,73 @@
 # Use the official Ruby image because the Rails images have been deprecated
-FROM ruby:2.5
+FROM ruby:2.5-slim
 
-# Enable https
-RUN apt-get update
-RUN apt-get install -y apt-transport-https
+# Enable package fetch over https and add a few core tools
+RUN apt-get update \
+    && apt-get install -y \
+       apt-transport-https \
+       curl \
+    && rm -rf /var/lib/apt/lists/*
 
 # Install Postgres client
-RUN apt-get install -y --no-install-recommends postgresql-client
-RUN rm -rf /var/lib/apt/lists/*
-
-# Install Chrome for capybara
```

Comment:
I noticed the Chromium bit didn't make it back in.  
If we want to run the full test suite _inside_ the container we'll need something like this.

Reply:
Maybe there are better ways?